scrapers/covid-19/historical.js

```diff
@@ -90,6 +137,13 @@ const historicalV2 = async (keys, redis) => {
  */
 const getHistoricalDataV2 = (data, lastdays = 30) => {
 	lastdays = stringUtils.getLastDays(lastdays);
+	// Removing data inserted for edge case of Canada
+	for (let i = 0; i < data.length; i++) {
+		if (data[i].country.toLowerCase() === 'canada' && data[i].province.toLowerCase() === 'recovered-aggregate') {
+			data.splice(i, 1);
+			break;
+		}
```

Comment:
this code is repeated, can you put it in a function called `removeReoveredAggregateCanada` or something? Would just clean up the code a bit

Reply:
Sure. Will do.